### DISCUSSION TITLE
Can someone tell me what does each encoder type setting stand for?

### DISCUSSION THREAD

**Discussion by michaellin99999** [coqui-ai/TTS#1108] [General Q&A]
1. gated_conv    = gated convolution? 
2. residual_conv_bn    = residual convolution batch normalization?
3. time_depth_separable  = time depth separable convolution...?
4. rel_pos_transformers. = what does rel, pos mean? what is this refering to?

**Comment by erogol** [comment]
"relative positional transformers. "

the rest is as you said.

**Reply by michaellin99999** [reply]
thanks
